arrows/core/render_mesh_depth_map.cxx

```diff
@@ -91,6 +96,7 @@ vital::image_container_sptr render_mesh_depth_map(vital::mesh_sptr mesh, vital::
 }
 
 
+/// This function renders a height map of a triangular mesh
 vital::image_container_sptr render_mesh_height_map(vital::mesh_sptr mesh, vital::camera_sptr camera)
 {
   vital::image_of<double> height_map(camera->image_width(), camera->image_height(), 1);
```

Comment:
You could initialize `height_map` with `transform_image` as you did in `render_mesh_depth_map`.